.github/workflows/acceptance-workflow.yml

```diff
@@ -69,14 +70,14 @@ jobs:
       - name: Install hedera local
         run: npm install @hashgraph/hedera-local -g
 
-      - name: Set operator id and key env variable if CI is manual
+      - name: Set operator id env variable if CI is manual
         if: ${{ inputs.operator_id }}
-        run: |
-          echo "OPERATOR_ID_MAIN=${{ inputs.operator_id }}" >> $GITHUB_ENV
-          echo "OPERATOR_KEY_MAIN=${{ inputs.operator_key }}" >> $GITHUB_ENV
+        run: echo "OPERATOR_ID_MAIN=${{ inputs.operator_id }}" >> $GITHUB_ENV
```

Comment:
This should be set as an `env:` section at the job or workflow level. Step and manual writing to the `$GITHUB_ENV` file is unnecessary.